internal/lsp/server.go

```diff
@@ -1631,7 +1626,25 @@ func (l *LanguageServer) handleTextDocumentCodeLens(
 		return nil, nil // return a null response, as per the spec
 	}
 
-	return rego.CodeLenses(ctx, params.TextDocument.URI, contents, module) //nolint:wrapcheck
+	ls, err := rego.CodeLenses(ctx, params.TextDocument.URI, contents, module)
+	if err != nil {
+		return nil, err
+	}
+
+	if l.clientInitializationOptions.EnableDebugCodelens != nil &&
+		*l.clientInitializationOptions.EnableDebugCodelens {
+		return ls, nil
+	}
+
+	// filter out `regal.debug` codelens
+	lenses := make([]types.CodeLens, 0, len(ls))
+	for _, lens := range ls {
```

Comment:
The linter issue here is basically just saying that these two lines aren't immediately related, and as such shouldn't be connected. So just add a newline in between, e.g:

```go
// filter out `regal.debug` codelens
lenses := make([]types.CodeLens, 0, len(ls))
	
for _, lens := range ls {
```